### PR TITLE
Remove Consultation presentation

### PR DESCRIPTION
**Pull request by andrewgarner**
Consultations have been migrated and are now rendered by [Government Frontend](alphagov/government-frontend). This removes the frontend code that is no longer needed.

[Trello](https://trello.com/c/LYFL1ZH9/)

### CODE REVIEW

File: app/controllers/consultations_controller.rb [hunk 1]
```diff
@@ -1,18 +0,0 @@
-class ConsultationsController < DocumentsController
-  def index
-    filter_params = params.except(:controller, :action, :format, :_)
```
Comment (fofr) [inline app/controllers/consultations_controller.rb]: Isn't this still needed? I think it's responsible for this redirect: https://www.gov.uk/government/consultations

Comment (andrewgarner) [inline app/controllers/consultations_controller.rb]: It is. Good spot.

Comment (andrewgarner) [inline app/controllers/consultations_controller.rb]: Should be fixed with 53ddfb7

**Comment by andrewgarner** [comment]
We can merge this once the 4 unpublishing routes that currently point to whitehall are addressed.

File: app/controllers/consultations_controller.rb [hunk 2]
```diff
@@ -3,16 +3,4 @@ def index
     filter_params = params.except(:controller, :action, :format, :_)
     redirect_to publications_path(filter_params.merge(publication_filter_option: 'consultations'))
```
Comment (gpeng) [inline app/controllers/consultations_controller.rb]: It's a shame to have to leave this in. We could just add the redirect into the router. That might be a bit 'magic' though. Thoughts?

Comment (fofr) [inline app/controllers/consultations_controller.rb]: I'd prefer it to be in the router. That's where I'd go to look for redirects.

Comment (andrewgarner) [inline app/controllers/consultations_controller.rb]: Can we do merging of `filter_params` in the router? Do we still need to?

Comment (gpeng) [inline app/controllers/consultations_controller.rb]: Could we use a prefix route? `/government/consultations`. That might let the filter params pass through to Whitehall with the explicit routes taking the actual consultation requests to government frontend. Need to check that though as I'm not sure.

Comment (andrewgarner) [inline app/controllers/consultations_controller.rb]: Going to move this and look to address the index action in a subsequent PR.

**Pull request merged**